### PR TITLE
Added jenkins connector

### PR DESCRIPTION
**Pull request by rameshlk**

### CODE REVIEW

File: basic_auth/jenkins.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      },
+      {
+        name: 'username',
+        optional: true,
+        hint: 'Your username'
+      },
+      {
+        name: 'password',
+        control_type: 'password',
+        label: 'Password or API key',
```
Comment (eeshan92) [inline basic_auth/jenkins.rb]: Should not be 'Password or API Key' and should be optional: false

File: basic_auth/jenkins.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
+{
+  title: 'Jenkins',
+
+  connection: {
+    fields: [
+      {
+        name: 'subdomain',
+        control_type: 'subdomain',
+        hint: 'Your port address'
```
Comment (eeshan92) [inline basic_auth/jenkins.rb]: should be optional: false

File: basic_auth/jenkins.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      },
+      {
+        name: 'username',
+        optional: true,
```
Comment (eeshan92) [inline basic_auth/jenkins.rb]: should be optional: false

File: basic_auth/jenkins.rb [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    get("http://#{connection['subdomain']}/asynchPeople/api/json")
+  },
+
+  actions: {
+    create_build: {
+
+      description: 'Create <span class="provider"> build </span> in <span class="provider">Jenkins</span>',
+
+      input_fields: ->() {[
+        { name: 'job', label: 'JOB NAME', optional: false}]
```
Comment (eeshan92) [inline basic_auth/jenkins.rb]: fix label

File: basic_auth/jenkins.rb [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+      },
+      },
+  },
+
+  triggers: {
+    new_job: {
+
+      description: 'New <span class="provider"> job </span> in <span class="provider">Jenkins</span>',
+
+      input_fields: ->() {
```
Comment (eeshan92) [inline basic_auth/jenkins.rb]: possible to take a last_updated parameter?

File: basic_auth/jenkins.rb (hunk 1 above)
Comment (rameshlk) [inline basic_auth/jenkins.rb]: We can use Password or api key.

**Pull request closed without merging**